scripts/Jenkinsfile

```diff
@@ -1,43 +1,48 @@
 pipeline 
 {
 
-    agent 
-    {
+    agent{
         node 
         {
             label 'qeff_node'
         }
     }
-
+    environment 
+    {
+        CONTAINER_NAME = "qeff_container"
+    }
     stages 
     {
+        
         stage('Install QEfficient')
         {
             steps
             {
                 sh '''
-                    python3.8 -m venv preflight_qeff
-                    . preflight_qeff/bin/activate
-                    pip install --upgrade pip setuptools
-                    pip install .[test]
-                    rm -rf QEfficient
+                    docker run --privileged -dit --name ${CONTAINER_NAME} -v ./:/data -v  /home/ubuntu/.cache/huggingface/hub:/root/.cache/huggingface/hub docker-registry.qualcomm.com/qraniumtest/qranium:master_latest
```

Comment:
Let's use ENV variables for mounting directories, do not expose directories here.